media/js/firefox/accounts.js

```diff
@@ -20,6 +24,18 @@
         fxaIframeSrc = fxaIframeSrc.replace('context=iframe', 'context=fx_firstrun_v2');
     }
 
+    // check for fxaEmail in sessionStorage or URL params
+    try {
+        fxaEmail = sessionStorage.getItem('fxa-email');
+    } catch(ex) {
+        fxaEmail = params.get('fxa-email');
```

Comment:
Sounds good :+1: I would favor relying on session storage as a hard-dependency, rather than falling back to reading via query param.